orbital_core/app.py

```diff
@@ -1,25 +1,37 @@
 from .routes import add_routes
-from aiohttp import ClientSession
+from .utils.tracing import configure_tracing, get_trace_config
+from .session import ExtendedClientSession
+
 
 def bootstrap_app(app, root_dir=None,
                   service_name="my service",
-                  service_description="this is a service."):
+                  service_description="this is a service.",
+                  tracing_route_blacklist=None,
```

Comment:
to help documentation, can type annotations be added? or documentation in the docstring? https://docs.python.org/3/library/typing.html

Reply:
I have added type annotations.